CMakeLists.txt

```diff
@@ -487,6 +487,9 @@ set(tests_root ${CMAKE_CURRENT_BINARY_DIR}/src/tests)
 
 
 if(opencoarrays_aware_compiler)
+  if (RUN_DEVELOPER_TESTS)
+    message ( STATUS "Running Developer tests is enabled." )
+  endif()
```

Comment:
Proposing this was more a way of not losing the message, then to put it in the right location.